querybook/webapp/components/DataDocRightSidebar/DataDocCloneButtonConfirm.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+
+import { availableEnvironmentsSelector } from 'redux/environment/selector';
+import { IEnvironment } from 'redux/environment/types';
+import { makeSelectOptions, Select } from 'ui/Select/Select';
+import { StyledText } from 'ui/StyledText/StyledText';
+
+interface IProps {
+    defaultEnvironment: IEnvironment;
+    onEnvironmentChange: (environment: IEnvironment) => void;
+}
+
+export const DataDocCloneButtonConfirm: React.FunctionComponent<IProps> = ({
```

Comment:
this component name seems a bit weird to me. I think this component could be just an environment selector, and move the messages to the DataDocCloneButton component.